src/ui/views/SettingsView/index.tsx

```diff
@@ -1,10 +1,12 @@
 import { Card, CardContent, Divider, Typography } from '@mui/material';
 import React, { FunctionComponent } from 'react';
 import SettingsIcon from '@mui/icons-material/Settings';
+import { useTranslation } from 'react-i18next';
 import CardTitle from '../../components/CardTitle';
 import MainLayout from '../../layouts/MainLayout';
 
 const SettingsView: FunctionComponent = () => {
```

Comment:
I think we should make the Settings page visible and allow our users to override the current language of the ExpressLRS Configurator.